Sources/SwifterSwift/SwiftStdlib/ArrayExtensions.swift

```diff
@@ -122,7 +122,7 @@ public extension Array where Element: Equatable {
     /// - Returns: an array of unique elements.
     func withoutDuplicates<E: Equatable>(keyPath path: KeyPath<Element, E>) -> [Element] {
         return reduce(into: [Element]()) { (result, element) in
-            if !result.contains { $0[keyPath: path] == element[keyPath: path] } {
+            if !result.contains(where: { $0[keyPath: path] == element[keyPath: path] }) {
```

Comment:
Why this change?

Reply:
SwiftLint complains if you have nested trailing closures